api/controllers/auth-controller.js

```diff
@@ -47,7 +48,7 @@ const signIn = async (req, res) => {
         if (!validPassword) return res.status(400).json({ message: 'Invalid email or password.'});
 
         const token = user.generateAuthToken();
-        res.status(200).send(token);
+        res.status(200).send({token, user});
```

Comment:
should be the same logic as for signup. Move token to headers and user to body
![image](https://user-images.githubusercontent.com/12981061/52791482-a9635900-302e-11e9-8183-5caadf8e277b.png)